core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -69,19 +71,24 @@ public Table load(String location) {
    * location.
    *
    * @param schema iceberg schema used to create the table
-   * @param spec partition specification
+   * @param spec partition specification. It can be null in case of unpartitioned table
+   * @param properties properties of the table to be created, it can be null
```

Comment:
How about "a string map of table properties, initialized to empty if null"

Reply:
Done